### PR TITLE
docs for Squeezebox server device binary sensors

### PR DESCRIPTION
**Pull request by pssc**
## Proposed change

Update for Squeezebox adds Server device with binary sensors 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122473
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Enhanced documentation for the Squeezebox integration, now supporting both Lyrion Music Server and Squeezebox players.
	- Added support for sensor functionalities, enabling monitoring of the media server's status.

- **Documentation**
	- Improved clarity and structure of the integration documentation to reflect new capabilities and transitions.

### CODE REVIEW

File: source/_integrations/squeezebox.markdown [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Squeezebox (Lyrion Music Server)
-description: Instructions on how to integrate a Squeezebox player into Home Assistant.
+description: Instructions on how to integrate a Lyrion Music Server(LMS) and squeezebox players into Home Assistant.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
description: Instructions on how to integrate Squeezebox players and a Lyrion Music Server (LMS)  into Home Assistant.
```
As the integration is called Squeezebox, Squeezebox should probably be mentioned before the music server.

File: source/_integrations/squeezebox.markdown [hunk 2]
```diff
@@ -12,10 +12,13 @@ ha_config_flow: true
...
 ha_platforms:
   - media_player
+  - binary_sensor
 ha_integration_type: integration
 ---
 
-The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was previously known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was formerly known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+
+This integration provides both media players connected to the server and supporting sensors for the server status.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
This integration provides both media players connected to the server and supporting binary sensors for the server status.
```

File: source/_integrations/squeezebox.markdown [hunk 3]
```diff
@@ -12,10 +12,13 @@ ha_config_flow: true
 ha_dhcp: true
 ha_platforms:
   - media_player
+  - binary_sensor
 ha_integration_type: integration
 ---
 
-The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was previously known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+The Squeezebox integration allows you to control music players from the [Lyrion Music Server](https://lyrion.org/) (LMS) ecosystem.  Lyrion Music Server was formerly known as [Logitech Media Server](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29).
+
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown:20]: Further down, before the Action section, please add a section called ## Entities, ### Binary sensors, listing the binary sensors together with a description.

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]

File: source/_integrations/squeezebox.markdown [hunk 4]
```diff
@@ -46,6 +49,15 @@ transporter_toslink:
         media_content_type: "music"
 ```
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Sever Service would like to be restarted usually to apply updates.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]: what do you mean by "usually"? It needs to be restarted? It needs to be restarted on a regular basis?

Comment (pssc) [reply source/_integrations/squeezebox.markdown]: usually its for updates, so comma after restarted?

Comment (c0ffeeca7) [reply source/_integrations/squeezebox.markdown]: ah!

Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
  Server Service needs to be restarted (typically, this is needed to apply updates).
```
Maybe something like this?

File: source/_integrations/squeezebox.markdown [hunk 5]
```diff
@@ -46,6 +49,15 @@ transporter_toslink:
...
+- Library rescan
+  Music library is currently being scaned by LMS.
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]: does this have implications for me as a user? is some service not available during that period?

File: source/_integrations/squeezebox.markdown [hunk 6]
```diff
@@ -46,6 +49,15 @@ transporter_toslink:
...
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Server Service needs to be restarted (typically, this is needed to apply updates).
+- Library rescan
+  Music library is currently being scaned by LMS.
+
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown:60]: ## Actions

Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown:60]:
```suggestion

## Actions

```

File: source/_integrations/squeezebox.markdown [hunk 7]
```diff
@@ -46,6 +49,15 @@ transporter_toslink:
...
 ```
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Server Service needs to be restarted (typically, this is needed to apply updates).
+- Library rescan
+  Music library is currently being scaned by LMS (during a scan some content may be unavailable dependant on the type of scan).
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
```

File: source/_integrations/squeezebox.markdown [hunk 8]
```diff
@@ -46,6 +49,15 @@ transporter_toslink:
...
 ```
 
+## Entities
+
+### Binary sensors
+
+- Needs restart
+  Server Service needs to be restarted (typically, this is needed to apply updates).
+- Library rescan
+  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown]:
```suggestion
- **Needs restart**:  Server Service needs to be restarted (typically, this is needed to apply updates).
- **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
```

**Review by c0ffeeca7 — DISMISSED** [review]
Thank you, @pssc :+1:

File: source/_integrations/squeezebox.markdown [hunk 9]
```diff
@@ -46,6 +49,17 @@ transporter_toslink:
...
 
+## Entities
+
+### Binary sensors
+
+- **Needs restart**:  Server Service needs to be restarted (typically, this is needed to apply updates).
+- **Library rescan**:  The music library is currently being scanned by LMS (depending on the type of scan, some content may be unavailable).
+
+## Actions
+
```
Comment (c0ffeeca7) [inline source/_integrations/squeezebox.markdown:60]:
```suggestion
```

**Review by c0ffeeca7 — APPROVED** [review]

**Comment by pssc** [comment]
Thanks for the guidance on my first document commit @c0ffeeca7 its much better.

**Pull request merged**